.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Chromatic Publish'
+
+on: push
```

Comment:
let's limit this to only run when a PR gets merged to the main branch

@vuonga1103